GliaWidgets/SecureConversations/Confirmation/SecureConversations.ConfirmationView.swift

```diff
@@ -177,6 +177,7 @@ extension SecureConversations {
             checkMessagesButton.setTitleColor(props.style.checkMessagesButtonStyle.textColor, for: .normal)
             checkMessagesButton.backgroundColor = props.style.checkMessagesButtonStyle.backgroundColor
             checkMessagesButton.accessibilityTraits = .button
+            checkMessagesButton.accessibilityIdentifier = "secureConversations_confirmationCheckMessages_button"
```

Comment:
some accessibility classes have hints and labels, so perhaps we can create identifiers there or reuse accessibility hints?

Reply:
The accessibility identifiers are useful for acceptance tests, so it's better if they are in this format. The hints should be complete sentences, so they are weird if being re-used as accessibility identifiers